### PR TITLE
VSCode configurations

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Helping our devs to work with vscode! :)

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            "name": "Python: Upload AbuseDB",
+            "type": "python",
+            "request": "launch",
+            "module": "demisto_sdk",
+            "cwd": "${env:CONTENT}",
+            "args": [
+                "upload",
+                "-i",
+                "${env:content}/Packs/AbuseDB/Integrations/AbuseDB"
```
Comment (Shellyber) [inline .vscode/launch.json:13]:
```suggestion
                "${env:CONTENT}/Packs/AbuseDB/Integrations/AbuseDB"
```

**Review by Shellyber — COMMENTED** [review]
Nice Job 👍 
Please fix the comment

**Review by Shellyber — APPROVED** [review]

**Review by avidan-H — APPROVED** [review]
This whole PR is to fix the editor suggestions regarding spaces and imports?

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            "args": [
+                "lint", // Lint AbuseDB
+                "-i",
+                "${env:CONTENT}/Packs/AbuseDB/Integrations/AbuseDB",
+                "-j",
+                "${env:CONTENT}/.demisto-sdk-report" // Write reports
+            ]
+        }
+    ]
+}
```
Comment (roysagi) [inline .vscode/launch.json:32]: @jochman - here is your problem.
image

**Pull request merged**